src/Session.h

```diff
@@ -366,6 +366,11 @@ class Session {
   TicksSemantics ticks_semantics() const { return ticks_semantics_; }
 
   virtual int cpu_binding(TraceStream& trace) const;
+  int cpu_binding() {
+    if (auto stream = trace_stream())
+      return cpu_binding(*stream);
```

Comment:
We put {} around everything

Reply:
.... I thought I checked and fixed all the style mismatch but apparently not...